.gitignore

```diff
@@ -4,4 +4,3 @@ npm-debug.log.*
 htmldocs/
 lerna-debug.log
 lib/
-.DS_Store
```

Comment:
Why did you remove this? I see `.DS_Store` files in my untracked files all the time. I think this should be there prevent Mac users from inadvertently committing them—or save them from having to manually select the files they want to commit to avoid committing `.DS_Store` files.